### PR TITLE
Translated default strings

### PR DESCRIPTION
**Pull request by labunchemjong**
Translated default strings

### CODE REVIEW

**Review by kafleg — APPROVED** [review]
LGTM

File: patterns/cta.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (carolinan) [inline patterns/cta.php]:
```suggestion
```

**Comment by carolinan** [comment]
@labunchemjong Thank you. What is your WordPress.org username so that we can add you to the contributors list? :)

**Comment by kafleg** [comment]
https://profiles.wordpress.org/labunchemjong/ 

I can make PR for updating the contributors list.

**Pull request merged**